frontend/src/components/question-creation/question-elements/ShortAnswerElement.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Button, IconButton, Input, Flex, Box, Spacer } from "@chakra-ui/react";
+import { EditOutlineIcon } from "../../../assets/icons";
+import EditShortAnswerModal from "./modals/short-answer/EditShortAnswerModal";
+
+interface ShortAnswerElementProps {
+  id: string;
+  data: string;
```

Comment:
is there a reason we store it as a string instead of number?

Reply:
also inputs such as `01` are allowed, not sure how we want to handle that

Reply:
updated to store as number! should resolve the `01` issue.